package.json

```
@@ -10,6 +10,8 @@
     "lint": "eslint --cache --fix .",
     "lint:check": "eslint --cache .",
     "lint:links": "iota-wiki check",
+    "lint:md": "yarn iota-wiki lint --fix",
```

```suggestion
    "lint:md": "yarn iota-wiki lint --fix docs/**/*.{md,mdx}",
```

Let's keep these in sync, just like the other commands either check or fix the same thing.